doc/api/url.md

```diff
@@ -254,7 +254,7 @@ console.log(myURL.hostname);
 // Setting the hostname does not change the port
```

Comment:
```suggestion
// Setting an invalid hostname is ignored:
```

Reply:
Applied your suggestion. Thank you for suggestion.